util/importers/LDC97S62.py

```diff
@@ -0,0 +1,363 @@
+                                                  capacity = 2 * self._get_device_count() * batch_size)
+        self._enqueue_op = self._example_queue.enqueue([self._x, self._x_length, self._y, self._y_length])
+        self._txt_files = txt_files
+        self._batch_size = batch_size
+        self._numcontext = numcontext
+        self._thread_count = thread_count
+        self._files_circular_list = self._create_files_circular_list()
+
+    def _get_device_count(self):
+        available_gpus = get_available_gpus()
+        return  max(len(available_gpus), 1)
+
+    def start_queue_threads(self, session):
+        batch_threads = [Thread(target=self._populate_batch_queue, args=(session,)) for i in xrange(self._thread_count)]
+        for batch_thread in batch_threads:
+            batch_thread.daemon = True
+            batch_thread.start()
+        return batch_threads
+
+    def _create_files_circular_list(self):
+        priorityQueue = PriorityQueue()
+        for txt_file in self._txt_files:
+            wav_file = os.path.splitext(txt_file)[0] + ".wav"
+            wav_file_size = os.path.getsize(wav_file)
+            priorityQueue.put((wav_file_size, (txt_file, wav_file)))
+        files_list = []
+        while not priorityQueue.empty():
+            priority, (txt_file, wav_file) = priorityQueue.get()
+            files_list.append((txt_file, wav_file))
+        return cycle(files_list)
+
+    def _populate_batch_queue(self, session):
+        for txt_file, wav_file in self._files_circular_list:
+            source = audiofile_to_input_vector(wav_file, self._numcep, self._numcontext)
+            source_len = len(source)
+            with codecs.open(txt_file, encoding="utf-8") as open_txt_file:
+                target = unicodedata.normalize("NFKD", open_txt_file.read()).encode("ascii", "ignore")
+                target = text_to_char_array(target)
+            target_len = len(target)
+            try:
+                session.run(self._enqueue_op, feed_dict={
+                    self._x: source,
+                    self._x_length: source_len,
+                    self._y: target,
+                    self._y_length: target_len})
+            except (RuntimeError, tf.errors.CancelledError):
+                return
+
+    def next_batch(self):
+        source, source_lengths, target, target_lengths = self._example_queue.dequeue_many(self._batch_size)
+        sparse_labels = ctc_label_dense_to_sparse(target, target_lengths, self._batch_size)
+        return source, source_lengths, sparse_labels
+
+    @property
+    def total_batches(self):
+        # Note: If len(_txt_files) % _batch_size != 0, this re-uses initial _txt_files
+        return int(ceil(float(len(self._txt_files)) / float(self._batch_size)))
+
+
+def read_data_sets(data_dir, train_batch_size, dev_batch_size, test_batch_size, numcep, numcontext, thread_count=8, limit_dev=0, limit_test=0, limit_train=0):
```

Comment:
Sorry but you said this a month ago so I am confused:  https://github.com/mozilla/DeepSpeech/pull/121#issuecomment-260991782